tests/test_commands_cache.py

```diff
@@ -173,3 +173,31 @@ def test_non_exist(self, xession):
     def test_bash_and_is_alias_is_only_functional_alias(self, xession):
         xession.aliases["git"] = lambda args: os.chdir(args[0])
         assert xession.commands_cache.is_only_functional_alias("git") is False
+
+
+def test_update_cache(xession, tmp_path):
+    # run test only if the cache is disabled
+    if xession.env.get("ENABLE_COMMANDS_CACHE"):
```

Comment:
the test should set the variable to False and proceed. You can see the other tests in this file for similar tests.

Reply:
done